api/Window.json

```
@@ -4395,10 +4395,12 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "From Firefox 82, <code>window.name</code> is reset if a new page from another domain is loaded into a tab. See <a href='https://bugzil.la/444222'>bug 444222</a>."
```

I'm a little concerned that this note reads as if this is unintended behavior, plus I'd like to be more explicit about what "reset" means. I'm open to further edits, but what to do you think of this?

```suggestion
              "notes": "From Firefox 82, if a new page from another domain is loaded into a tab, then <code>window.name</code> is set to the empty string. This change prevents some cross-site attacks. See <a href='https://bugzil.la/444222'>bug 444222</a>."
```